android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt

```diff
@@ -110,9 +113,13 @@ class VideoPlaybackService : MediaSessionService() {
             return
         }
 
+        val returnToStartedActivity = Intent(this, startedFrom).apply {
```

Comment:
this is also a strange val name: returnToStartedActivity I would expect an Activity.
what do you think about: backToPlayerIntent ?

Reply:
(not mandatory, just a proposal)

Reply:
I had a tough time figuring out what to name these, thank you!

Reply:
@paulrinaldi no problem, it is the principle of code review. #IAmTheBadCop 👮‍♂️ ;)

Reply:
Done.